containers/datalab/content/GCSbackup.sh

```diff
@@ -123,7 +123,11 @@ fi
 # test and create bucket if necessary
 gsutil ls gs://${gcs_bucket} &>/dev/null || {
   echo "Bucket '${gcs_bucket}' was not found. Creating it.."
-  gsutil mb gs://"${gcs_bucket}"
+  gsutil mb gs://"${gcs_bucket}" || {
```

Comment:
Do you mean we shouldn't create the `gs://${project_id}.appspot.com` bucket if it doesn't exist? Other than this, I don't see much difference between the comment and the changes.